File: Examples/CSharp/Articles/ManagingWorkbooksWorksheets/ImportCSVWithFormulas.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Aspose.Cells.Examples.CSharp.Articles.ManagingWorkbooksWorksheets
{
    public class ImportCSVWithFormulas
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();

            TxtLoadOptions opts = new TxtLoadOptions();
            opts.Separator = ',';
            opts.HasFormula = true;

            // Load your csv file with formulas in a Workbook object
            Workbook workbook = new Workbook(sourceDir + "sampleImportCSVWithFormulas.csv", opts);

            // You can also import your csv file like this
            // The code below is importing csv file starting from cell D4
            Worksheet worksheet = workbook.Worksheets[0];
            worksheet.Cells.ImportCSV(sourceDir + "sampleImportCSVWithFormulas.csv", opts, 3, 3);

            // Save your workbook in Xlsx format
            workbook.Save(outputDir + "outputImportCSVWithFormulas.xlsx");

            Console.WriteLine("ImportCSVWithFormulas executed successfully.");
        }
    }
}

File: Examples/VisualBasic/Articles/ManagingWorkbooksWorksheets/ImportCSVWithFormulas.vb
Imports System.IO
Imports Aspose.Cells

Namespace Articles.ManagingWorkbooksWorksheets
    Public Class ImportCSVWithFormulas
        Public Shared Sub Run()
            ' ExStart:ImportCSVWithFormulas
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            Dim opts As New TxtLoadOptions()
            opts.Separator = ","c
            opts.HasFormula = True

            ' Load your CSV file with formulas in a Workbook object
            Dim workbook As New Workbook(dataDir & Convert.ToString("sample.csv"), opts)

            ' You can also import your CSV file like this
            ' The code below is importing CSV file starting from cell D4
            Dim worksheet As Worksheet = workbook.Worksheets(0)
            worksheet.Cells.ImportCSV(dataDir & Convert.ToString("sample.csv"), opts, 3, 3)

            ' Save your workbook in Xlsx format
            workbook.Save(dataDir & Convert.ToString("output_out_.xlsx"))
            ' ExEnd:ImportCSVWithFormulas
        End Sub
    End Class
End Namespace